test/api/Auth.js

```diff
@@ -189,4 +192,44 @@ describe('Auth', () => {
     });
   });
 
+  describe('checkIfPageCountResets()', () => {
+    it('Should reset user\'s page count when last login was over 7 days ago',
```

Comment:
let's skip using `\`s, we can rephrase the titles like
```suggestion
    it('Should reset page count when the last login was over 7 days ago',
```